src/styles/blocks/card.scss

```
@@ -0,0 +1,124 @@
+@mixin catalog-grid($columns) {
+  display: grid;
+  grid-template-columns: repeat($columns, $card-width);
+  gap: 46px 48px;
+  justify-content: center;
+}
+
+.cards {
+  padding-top: 50px;
+  padding-bottom: 50px;
+
+  @media (min-width: 488px) {
+    @include catalog-grid(2);
+  }
+
+  @media (min-width: 768px) {
+    @include catalog-grid(3);
+  }
+
+  @media (min-width: 1024px) {
+    @include catalog-grid(4);
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  margin-top: 0;
+
+  transition-property: transform, color, background;
+  transition-duration: $transition-duration;
+
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: $background-color;
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
```

reset default margin for `h2` and `p` tags (similar you did with `body`)
```suggestion
    margin-botom: 4px;
```